Python/Product/VSInterpreters/Interpreter/IInterpreterRegistryService.cs

```diff
@@ -84,5 +84,11 @@ public interface IInterpreterRegistryService {
         /// <see cref="BeginSuppressInterpretersChangedEvent"/>.
         /// </summary>
         void EndSuppressInterpretersChangedEvent();
+
+        /// <summary>
+        /// Discovers Conda interpreter factories after updating or installing 
+        /// a Python package.
+        /// </summary>
+        void DiscoverCondaInterpreterFactoriesAfterPythonInstall();
```

Comment:
This method name is too specific. Yes, you are calling it after a python install, but you could potentially call it for other reasons as well. Consider renaming this to `RefreshCondaInterpreterFactories()` or something similar.

Reply:
@AdamYoblick I've renamed it with what you suggested and tested it again.😊